src/mlserver_services/stable_diffusion/client.py

```diff
@@ -0,0 +1,25 @@
+from threading import Thread
+
+import requests
+from mlserver.codecs import StringCodec
+from mlserver.types import InferenceRequest
+
+endpoint = "http://localhost:8080/v2/models/stable-diffusion/infer"
```

Comment:
```suggestion
endpoint = "http://127.0.0.1:8080/v2/models/stable-diffusion/infer"
```

avoid resolving it to an IPv6 address